cypress/integration/users/create-users.spec.js

```diff
@@ -0,0 +1,19 @@
+import {
+  Pane,
+} from "../../../interactors";
+
+describe('Creating user', () => {
+  beforeEach(() => {
+    cy.login('diku_admin', 'admin');
+  });
+
+  it('should be possible', function () {
+    cy.visit('/users/create');
+    cy.createUser('Test123', 'undergrad (Undergraduate Student)', 'test@folio.org');
```

Comment:
`undergrad (Undergraduate Student)` name can't be constant - firstly fetch available patron groups via API and pass one of their names to form